Makefile

```diff
@@ -10,6 +10,8 @@ UPGRADE_HOOK_IMG := $(shell jq -r .version_upgrade_hook_image < $(MONGODB_COMMUN
 READINESS_PROBE_IMG := $(shell jq -r .readiness_probe_image < $(MONGODB_COMMUNITY_CONFIG))
 REGISTRY := $(shell jq -r .repo_url < $(MONGODB_COMMUNITY_CONFIG))
 AGENT_IMAGE_NAME := $(shell jq -r .agent_image_ubuntu < $(MONGODB_COMMUNITY_CONFIG))
+TEST_NAMESPACE := $(shell jq -r .namespace < $(MONGODB_COMMUNITY_CONFIG))
```

Comment:
it could be just TEST_NAMESPACE = $(NAMESPACE)

Reply:
we could just use NAMESPACE and get rid of TEST_NAMESPACE.